### PR TITLE
Fundamentals package

### LINKED ISSUE #9645: office-ui-fabric-react package breakup
In an effort to ensure maintenance and development in
office-ui-fabric-react continues to be sustainable, we are proposing
the task of breaking out controls into smaller packages within this
repo. These packages will be consumed and re-exported by
`office-ui-fabric-react` as to not change the API surface of
`office-ui-fabric-react` itself.

Doing this work has several benefits:
- Allow packages to be deveoped in standalone packages without
  introducing circular dependencies when releasing.
- Allow packages to release versions at different cadences. Targeted
  hotfixes can be picked up by groups at their own need while other
  packages are free to iterate at their own pace.
- Gives us options of moving chunks of code to separate repositories
  more easily in the future.
- Creates framework for allowing larger ecosystem of high value-add
  controls.

The specific packages will contain the following controls:

| @uifabric/$PACKAGE | controls |
|--------------------|----------|
| react-fundamentals | `ActivityItem`, `DocumentCard`, `HoverCard`, `Breadcrumb`, `CommandBar`, `Nav`, `OverflowSet`, `Pivot`, `ResizeGroup`, `ContextualMenu`, `Announced`, `Color`, `Divider`, `Grid`, `Icon`, `Icons`, `Image`, `Keytip`, `KeytipData`, `KeytipLayer`, `Link`, `Separator`, `Stack`, `Text`, `Callout`, `Dialog`, `Layer`, `Modal`, `Overlay`, `Panel`, `Popup`, `ScrollablePane`, `Sticky`, `Tooltip`, `FocusTrapZone`, `FocusZone`, `Autofill`, `Button`, `Checkbox`, `ChoiceGroup`, `ChoiceGroupOption`, `ComboBox`, `Dropdown`, `Label`, `Rating`, `SearchBox`, `SelectableOption`, `Slider`, `SpinButton`, `TextField`, `Toggle`, `Coachmark`, `MessageBar`, `PositioningContainer`, `ProgressIndicator`, `Shimmer`, `Spinner`, `TeachingBubble` |
| react-data-views | `Check`, `DetailsList`, `GroupedList`, `List`, `MarqueeSelection`, `SelectedItemsList`, `ShimmeredDetailsList` |
| react-date-time | `Calendar`, `DatePicker` |
| react-people | `Facepile`, `Persona`, `PersonaCoin`, `PersonaPresence` |
| react-pickers | `ColorPicker`, `ExtendedPicker`, `FloatingPicker`, `Pickers`, `SwatchColorPicker` |

Work will be staged 1 package at a time, moving necessary controls from `office-ui-fabric-react` then publishing a minor release. This will ensure that pull requests are reviewable and releasable at regular intervals.

Please add comments!

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #9645
- [X] Include a change request file using `$ npm run change`

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9859)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/packages/react-date-time/components/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { styled } from '../../../../Utilities';
+import { IDatePickerProps, IDatePickerStyleProps, IDatePickerStyles } from './DatePicker.types';
+import { DatePickerBase } from './DatePicker.base';
+import { styles } from './DatePicker.styles';
+
+/**
+ * DatePicker description
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/packages/react-date-time/components/DatePicker/DatePicker.tsx]: Not related to this PR, just found this really funny 😆

File: packages/office-ui-fabric-react/src/packages/react-fundamentals/Layer.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import '../../version';
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/packages/react-fundamentals/Layer.ts]: I wonder, do we need this? It isn't in any other export file.

File: packages/office-ui-fabric-react/src/packages/react-fundamentals/components/Button/index.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export * from './Button';
+export * from './ActionButton/ActionButton';
+export * from './CommandBarButton/CommandBarButton';
+export * from './CommandButton/CommandButton';
+export * from './CompoundButton/CompoundButton';
+export * from './DefaultButton/DefaultButton';
+export * from './MessageBarButton/MessageBarButton';
+export * from './PrimaryButton/PrimaryButton';
+export * from './IconButton/IconButton';
+export * from './BaseButton.classNames'; // TODO removeme
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/packages/react-fundamentals/components/Button/index.ts]: Adding comment to track todo.

**Comment by jdhuntington** [comment]
Closing as the build isn't running from this PR. Creating a new branch and reopening.

**Pull request closed without merging**